src/main/java/uk/gov/pay/directdebit/mandate/model/MandateState.java

```diff
@@ -19,7 +19,8 @@
     PENDING(EXTERNAL_PENDING),
     ACTIVE(EXTERNAL_ACTIVE),
     FAILED(EXTERNAL_INACTIVE),
-    CANCELLED(EXTERNAL_INACTIVE);
+    CANCELLED(EXTERNAL_INACTIVE),
```

Comment:
I didn't add cancelled, I added expired, however it looks like cancelled may still be being used even if not explicitly declared since updateMandate will look for the next status based on the current status and the SupportedEvent type. Here is an example from MandateService, if the mandate is in pending or active it'll be moved to cancelled.
```
public DirectDebitEvent mandateCancelledFor(Mandate mandate) {
        Mandate newMandate = updateStateFor(mandate, MANDATE_CANCELLED);
        userNotificationService.sendMandateCancelledEmailFor(newMandate);
        return directDebitEventService.registerMandateCancelledEventFor(newMandate);
    }
```

Reply:
ah ok. i read it wrong. the line changed because of the comma, not because you added it!